yellow-paper/docs/decentralization/actors.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Nodes and Actors
+---
+
+To analyze the suitability of different node types for various actor types in your system, let's break down the node types and actor types, considering the characteristics, preferences, abilities, and potential motivations of each actor. This will help determine which actors are likely to run which nodes and how they might interact with the Private Execution Environment (PXE).
```

Comment:
```suggestion
To analyze the suitability of different node types for various actor types in your system, let's break down the node types and actor types, considering the characteristics, preferences, abilities, and potential motivations of each actor. This will help determine which actors are likely to run which nodes and how they might interact with the Private eXecution Environment (PXE).
```

😝